src/content/blog/astro-cloudcannon-support.mdx

```diff
@@ -0,0 +1,37 @@
+publishDate: "Mar 17, 2023"
+authors:
+  - thuy
+coverImage: "/src/content/blog/_images/astro-cloudcannon-hero.jpeg"
+socialImage: "/src/content/blog/_images/astro-cloudcannon-social.jpeg"
+lang: "en"
+---
+
+[CloudCannon](https://cloudcannon.com/astro-cms/?utm_campaign=Astro%20out-of-beta&utm_source=astro-partner-blog&utm_medium=blog) — a popular Git-based CMS for static site generators — has announced full support for 
+Astro, with live visual editing ready to set up for all Astro sites.
+
+With this release, Astro users can design and configure their ideal editing interface for new and 
+existing Astro projects within CloudCannon, with options available for intuitive no-code interfaces 
+for non-technical teams, as well as publishing and reviewing workflows for large organizations.
+
+To get started with CloudCannon as your Astro CMS, create a CloudCannon account and follow the 
+instructions at their [Astro Starter Guide](https://cloudcannon.com/documentation/guides/astro-starter-guide/?utm_campaign=Astro%20out-of-beta&utm_source=astro-partner-blog). In brief, you’ll need to:
+
+- Sync your Astro site files from GitHub, GitLab, or Bitbucket. Any changes you make in CloudCannon 
+will be pushed back to your Astro repository.
+- Build and configure your site. (Note that [CloudCannon Collections](https://cloudcannon.com/documentation/articles/defining-your-collections/?utm_campaign=Astro%20out-of-beta&utm_source=astro-partner-blog) correspond neatly to Astro 
+Content Collections.) You can also configure your inputs and structures, which will let you create 
+page builders or configurable navigations for your sites.
+- Set up visual editing. With CloudCannon’s [visual data previews](https://cloudcannon.com/documentation/articles/using-live-editing-with-vanilla-js/?utm_campaign=Astro%20out-of-beta&utm_source=astro-partner-blog) and Astro’s client-side hydration 
+you can listen for changes in CloudCannon and update your content on the page automatically. 
+Tagging elements with CloudCannon’s visual data bindings allows you to attach a UI directly to the 
+elements on the page within the CMS’s Visual Editor, giving site users an intuitive editing interface.
+
+For Astro users with MDX components, CloudCannon supports [embedding rich Snippets in Markdown 
+content](https://cloudcannon.com/documentation/articles/editing-with-mdx-components/?utm_campaign=Astro%20out-of-beta&utm_source=astro-partner-blog) — allowing content editors to use structured components without touching the underlying code.
+
+Finally, CloudCannon has also released a stylish new Astro theme, [Sendit](https://cloudcannon.com/templates/sendit/?utm_campaign=Astro%20out-of-beta&utm_source=astro-partner-blog), which is preconfigured 
+for live visual editing. It includes twenty pre-styled components, and is available both in the 
+[Astro Themes directory](https://astro.build/themes/details/sendit/), and for immediate deployment on CloudCannon.
```

Comment:
Nit: what do you think about reworking this sentence a bit to remove the multiples "and" in the same sentence?
```suggestion
Finally, CloudCannon has also released a stylish new Astro theme, [Sendit](https://cloudcannon.com/templates/sendit/?utm_campaign=Astro%20out-of-beta&utm_source=astro-partner-blog), which is preconfigured 
for live visual editing. It includes twenty pre-styled components, available both in the 
[Astro Themes directory](https://astro.build/themes/details/sendit/), and for immediate deployment on CloudCannon.
```